charts/astronomer/templates/houston/worker/houston-worker-deployment.yaml

```diff
@@ -83,19 +83,23 @@ spec:
         - name: houston
           image: {{ template "houston.image" . }}
           imagePullPolicy: {{ .Values.images.houston.pullPolicy }}
+          {{- if .Values.houston.workerCommand }}
```

Comment:
There is already a `houston.worker` namespace, though there is no `houston.api` namespace. I kinda think this should be `houston.worker.command`, `houston.worker.args`, and the same for `houston.api`. WDYT?

Reply:
yea i think it should be houston.worker.X